wireapp/wire-ios-mono
# feat: Sort conversations list search results - WPB-14955

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-14955" title="WPB-14955" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-14955</a>  Improve Search in all Conversations
  </td></table>
  <br />
 





### Issue

This PR adds sorting to conversation list search results so that:

- Matched one-on-one conversations appear at the beginning.
- Conversations where the query matches then name appear in the middle.
- Conversations where the query matches a participant name appear at the end.

### Testing

1. Navigate to the conversations tab.
2. Begin searching
3. Verify, that one-on-one conversations appear first, conversations where the query matches the conversation name appear next, and conversations where the query matches a participant name appear last.  

---

### Checklist
